.idea/codeStyles/Project.xml

```diff
@@ -3,7 +3,16 @@
     <JetCodeStyleSettings>
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value>
-          <package name="kotlinx.android.synthetic" withSubpackages="true" static="false" />
+          <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />
```

Comment:
Did you mean to change this ?

Reply:
Oh yes, my AS was keep changing that for some time and I needed to commit it. It becomes very hard for me to exclude these changes on each PR so if it is not affecting you in any way I would rather keep it. I wonder if @0xnm doesn't have same issue ?

Reply:
I'm on 4.1.2 and AS doesn't apply this change, but yeah, let's keep it if AS insists.

Reply:
Oh, that explains it, I am on 4.1.3

Reply:
Will add a task for it.